tools/cli/datalab.py

```diff
@@ -93,6 +98,10 @@
 environment variable CLOUDSDK_COMPUTE_ZONE.
 """)
 
+_EMAIL_HELP = ("""The email address associated with the instance.
+
+This should represent the user of Datalab.""")
```

Comment:
I think we should add something about using the email for git here, otherwise it's not clear why this is suddenly required